### PR TITLE
Add whitespace updates from new jupytext version

### PR DESCRIPTION
**Pull request by joelostblom**
Not the most important PR but I noticed these whitespace updates. Probably a result from a new version of jupytext?

### CODE REVIEW

File: source/classification1.md [hunk 1]
```diff
@@ -5,15 +5,14 @@ jupytext:
...
-    jupytext_version: 1.13.7
+    jupytext_version: 1.14.7
 kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
 ```{code-cell} ipython3
-:tags: [remove-cell]
```
Comment (trevorcampbell) [inline source/classification1.md]: Definitely should still be removing this cell

**Review by trevorcampbell — COMMENTED** [review]
@joelostblom I'm not sure what this PR is. I guess you're editing things as jupyter notebooks and then converting them to `.md` files, or something?

I don't see anything here I want to merge right now

File: source/classification1.md (hunk 1 above)
Comment (joelostblom) [reply source/classification1.md]: Yeah sorry that was me playing around, added it back

File: source/classification1.md [hunk 2]
```diff
@@ -5,15 +5,14 @@ jupytext:
...
     format_version: 0.13
-    jupytext_version: 1.13.7
+    jupytext_version: 1.14.7
 kernelspec:
   display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
 ```{code-cell} ipython3
```
Comment (joelostblom) [inline source/classification1.md]:
```suggestion
```{code-cell} ipython3
:tags: [remove-cell]
```

**Review by joelostblom — COMMENTED** [review]
Yes, these whitespace changes were made to the md files when I saved them after clicking "open as notebook" and editing the md files in jupyterlab. I mostly added them here because they will create noise in later PRs when we use jupyterlab/notebook to edit the md files.

**Comment by trevorcampbell** [comment]
Let's stick with editing `.md` files. I don't trust jupytext to properly track changes between formats without changing the compiled book, and I would prefer to avoid having to review the whole book for subtle changes each time someone decides to edit one or the other.

I vote we close this. If we want to do one final "sync" PR after our major edits are done to line up with the current jupytext version, I'm OK with that -- just not in the midst of editing.

**Pull request closed without merging**